server/api/batch_inventory.py

```diff
@@ -747,21 +755,23 @@ def complete_upload_for_batch_inventory_cvr(
     if not batch_inventory_data or not batch_inventory_data.system_type:
         raise Conflict("Must select system type before uploading CVR file.")
 
-    (storage_path, filename, file_type) = get_standard_file_upload_request_params(
-        request
-    )
-
-    if batch_inventory_data.system_type == CvrFileType.DOMINION:
-        validate_csv_mimetype(file_type)
-    elif batch_inventory_data.system_type == CvrFileType.ESS:
-        validate_csv_or_zip_mimetype(file_type)
+    expected_file_types = [FileType.CSV]
+    if batch_inventory_data.system_type == CvrFileType.ESS:
+        expected_file_types = [FileType.CSV, FileType.ZIP]
     elif batch_inventory_data.system_type == CvrFileType.HART:
-        validate_zip_mimetype(file_type)
-    else:
+        expected_file_types = [FileType.ZIP]
+    elif batch_inventory_data.system_type != CvrFileType.DOMINION:
```

Comment:
It's a little confusing that the dominion case is set as a default at the top and then checked as the last case in the if/else chain. Now that this is just a mapping from `CvrFileType` to `List[FileType]`, maybe we could just use a `dict`?